package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "runbook",
+  "name": "sr-runbook",
```

Comment:
This seems unrelated?

Reply:
This is `main` coming in I think

Reply:
Is your local directory called `sr-runbook` by chance?

Reply:
Ah yes, iunno how this got in, will fix

Reply:
https://github.com/srobo/runbook/pull/176/commits/76d13a7adb5e555cfe8a2e15163ec2ae893b409a